### DISCUSSION TITLE
Skip TPM Check on Dynamic Update

### DISCUSSION THREAD

**Discussion by elajits** [AveYo/MediaCreationTool.bat#148] [General]
Hey AveYo! 

I cant get the latest Windows 11 DEV Insider Preview 25126.1000 update
to work with the Skip_TPM_Check_on_Dynamic_Update. v9+ for 25115+
i just getting error Download error - 0x80070002

Big thx for amazing tools and scripts :)

**Comment by AveYo** [comment]
You have some other problem, the script works just fine with 25126.
Have you tried [windows_update_refresh.bat](https://github.com/AveYo/MediaCreationTool.bat/blob/main/bypass11/windows_update_refresh.bat)? It can fix some common updating issues.

Do note that the microsoft servers are known to timeout in some regions, specially on update day.
Restart, try again.
Changing country or region settings to something else can jump-start downloading errors.
You should also check your date and time are set correctly.

**Comment by elajits** [comment]
Hey okay yea i have run the windows_update_refresh.bat and that did not help!
the system time and date are correct to my location here in sweden! :)
I will try change region and country thx :)

**Reply by AveYo** [reply]
I would just temporarily set it to US
It takes some time for the microsoft CDN regional mirrors to update

Tho there is a chance the script does not work for you (if it's hindered by a zealous antivirus).
Check `C:\$WINDOWS.~BT\Sources\Panther\setupact.log`  
it should have a line with: CmdLine = [/Product Server /Compat IgnoreWarning /MigrateDrivers All ...

**Comment by elajits** [comment]
i have Region default to United states and time format to (UTC+01:00 Amsterdam,Berlin,Bern,Rome,Stockholm,ect)
i use windows defender i tried disable it during update!

okay i will tale a look at the setupact.log file :)

**Comment by elajits** [comment]
i notice that my C:\$WINDOWS.~BT is empty there is no files in it :)

**Comment by elajits** [comment]
now it seems do be downloading the update after i removed Skip_TPM_Check_on_Dynamic_Update.cmd
and re enabled it! :)

**Comment by gmy77** [comment]
I hope to find updated way's... sooner or later this trick not continue to work... i think.
